bodo/tests/test_table_column_del.py

```diff
@@ -24,6 +24,8 @@
 )
 from bodo.utils.utils import is_expr
 
+pytestmark = pytest.mark.slow
```

Comment:
These tests take ~15 minutes in total on the PR CI machines. Considering they are compiler tests that don't change often, I marked as slow.